earth_enterprise/src/server/wsgi/serve/publish/publish_manager_helper.py

```diff
@@ -1289,6 +1311,29 @@ def _ListTargetPaths(self):
         " WHERE target_id IN (SELECT target_id FROM target_db_table)")
     return self.DbQuery(query_string)
 
+  # Finds the database that is the default stream for Earth Clients.
+  def _GetEcDefaultGeTargetPath(self):
+    """Gets target paths serving published databases.
+
+    Raises:
+      psycopg2.Error/Warning.
+    Returns:
+      list of tuples (target_path, target_id, serve_wms).
+    """
+    target_path_result = None
+    query_string = (
+        """SELECT target_table.target_path, publish_context_table.ec_default_ge
```

Comment:
Q: The `publish_context_table.ec_default_ge` value doesn't seem to be used in this function.  Do you need to query for it?